src/backend/distributed/connection/placement_connection.c

```diff
@@ -1175,6 +1177,18 @@ InitPlacementConnectionManagement(void)
 }
 
 
+/*
+ * ShouldUseCleanConnection returns whether we should force a new connection if
+ * existing connections accessed non-co-located placements on the workers.
+ */
+bool
+ShouldUseCleanConnection(void)
```

Comment:
Can you revert this name? It seems unrelated to the flag.

Reply:
Done. I left it as the original `UseConnectionPerPlacement()`.

I thought that once we renamed the flag `s/CONNECTION_PER_PLACEMENT/REQUIRE_CLEAN_CONNECTION/`, it makes sense to replace occurences of `s/ConnectionPerPlacement/CleanConnection/`.

I'm ok with either way

Reply:
This mostly just shows how poorly the CONNECTION_PER_PLACEMENT flag was named. These two concepts are unrelated, but had the same name.